src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -72,8 +73,14 @@ public enum OperationKind
         /// <summary>Indicates an <see cref="IEventReferenceOperation"/>.</summary>
         EventReference = 0x1e,
         /// <summary>Indicates an <see cref="IUnaryOperation"/>.</summary>
+        Unary = 0x1f,
```

Comment:
>Unary [](start = 8, length = 5)

This and other renaming changes don't seem directly related to the core purpose of the PR (Lazy issue) and has the unfortunate effect of updating many tests :-( 
The renaming would have been better as a separate PR in my opinion.